tox.ini

```diff
@@ -19,7 +19,7 @@ deps =
     coverage
     flake8
     mock
-    neuron-nightly
+    neuron
```

Comment:
I'd prefer keep on testing on nightly though, to find issues as early as possible

Reply:
I think we just found an issue then :D

Reply:
https://github.com/BlueBrain/BluePyOpt/actions/runs/2549275161

Reply:
Is it something we have to report to the nrn github ?

Reply:
Probably best. This is an error generator by a neuron header  file, right?

Reply:
The issue is coming from this line: https://github.com/BlueBrain/BluePyOpt/blob/cc5448730d73311150421a4bec0930315d39cf9b/examples/stochkv/mechanisms/StochKv3.mod#L110

Reply:
Ah Pramod answered: https://github.com/neuronsimulator/nrn/issues/1875

Reply:
As mentioned in the [BBP internal slack channel](https://bluebrainproject.slack.com/archives/C02UDSR3AAY/p1655995850804189), assuming this StochKv3.mod from BBP, I would copy the updated version from BBP models (e.g. [here](https://bbpgitlab.epfl.ch/hpc/sim/models/neocortex/-/blob/main/mod/v6/StochKv3.mod))

Would be happy to help if anything else is required.